### PR TITLE
Lottie

### PR DESCRIPTION
**Pull request by AngelaLee18**
LOTTIE

DESCRIPTION

In this branch I added Lottie and using a loader animation instead of activity indicator.

EVIDENCE

https://user-images.githubusercontent.com/109097550/190170218-87b3dec3-a12f-4db5-9cb9-0f1875bfbca9.mov

### CODE REVIEW

File: Ted Talk/Podfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  target 'Ted TalkTests' do
+    inherit! :search_paths
+    # Pods for testing
+  end
+
+  target 'Ted TalkUITests' do
+    # Pods for testing
+  end
+
+pod 'lottie-ios'
```
Comment (ezequielvelezglobant) [inline Ted Talk/Podfile]: move to line 9. In this way you will know that is for all Ted Talk target

**Review by ezequielvelezglobant — CHANGES_REQUESTED** [review]
Remember to fix the MVVM comments and merge those changes into this branch.

File: Ted Talk/Ted TalkTests/SearchBarAndPickerViewTest.swift [hunk 2]
```diff
@@ -30,11 +30,11 @@ class SearchBarAndPickerViewTest: XCTestCase {
         //
         var searchResult: [TedTalkData] = []
 
-        searchResult = sut.searchEnteredWord(searchText: "event", picker: "Event")
+        searchResult = sut.searchByFilter(searchText: "event", picker: "Event")
```
Comment (RafaBenaGlobant) [inline Ted Talk/Ted TalkTests/SearchBarAndPickerViewTest.swift]: Now ViewModelTedTalk, not DataManager, is responsible for 'searchByFilter' but the test is not reflecting it.

File: Ted Talk/Podfile (hunk 1 above)
Comment (AngelaLee18) [reply Ted Talk/Podfile]: fc1099a7604956dc603a1208cc4d948ae3893834

File: Ted Talk/Ted TalkTests/SearchBarAndPickerViewTest.swift (hunk 2 above)
Comment (AngelaLee18) [reply Ted Talk/Ted TalkTests/SearchBarAndPickerViewTest.swift]: Correted 
2f3b00a8f4ca04701f3758f9597315166cd17c32

File: Ted Talk/Podfile [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Uncomment the next line to define a global platform for your project
+# platform :ios, '9.0'
+
+target 'Ted Talk' do
+  # Comment the next line if you don't want to use dynamic frameworks
+  use_frameworks!
+
+  # Pods for Ted Talk
+	pod 'lottie-ios'
```
Comment (ezequielvelezglobant) [inline Ted Talk/Podfile]: remove extra tabulation.

File: Ted Talk/Ted Talk/View/TableViewController.swift [hunk 4]
```diff
@@ -6,90 +6,99 @@
...
+        tableView.isHidden = true
+        animationView.contentMode = .scaleAspectFit
+        animationView.loopMode = .loop
+        animationView.play()
+    }
+    
+    func bind() {
+        viewModel.loadTableView = {
+
+            DispatchQueue.main.asyncAfter(deadline: .now() + 2.0) {
```
Comment (ezequielvelezglobant) [inline Ted Talk/Ted Talk/View/TableViewController.swift]: there is an extra space and also a missing weak self

**Review by ezequielvelezglobant — CHANGES_REQUESTED** [review]

File: Ted Talk/Podfile (hunk 3 above)
Comment (AngelaLee18) [reply Ted Talk/Podfile]: a94ce78491b4e27284abacb47269c863317f5059

File: Ted Talk/Ted Talk/View/TableViewController.swift (hunk 4 above)
Comment (AngelaLee18) [reply Ted Talk/Ted Talk/View/TableViewController.swift]: a94ce78491b4e27284abacb47269c863317f5059

**Review by ezequielvelezglobant — APPROVED** [review]

**Review by RafaBenaGlobant — APPROVED** [review]

**Review by JMVarelaGlobant — APPROVED** [review]

**Pull request merged**